### PR TITLE
Bump ecmarkup dependency and subdependency to fix deprecation warning

### PR DESCRIPTION
**Pull request by timotheeguerin**
Bump `ecmarkup` dependency and run rush update --full to update dependency of dependency causing a deprecation warning to be logged.

image

Spec.html code changed but result looks the same.

image

### CODE REVIEW

**Comment by nguerrera** [comment]
Build failure looks like we picked up a dependency that didn't work with our third party notice generation code's search for a license:

```
Error: Cannot get license for /home/vsts/work/1/s/common/temp/node_modules/.pnpm/tr46@0.0.3/node_modules/tr46, license file not found.
```

**Comment by timotheeguerin** [comment]
> Build failure looks like we picked up a dependency that didn't work with our third party notice generation code's search for a license:
> 
> ```
> Error: Cannot get license for /home/vsts/work/1/s/common/temp/node_modules/.pnpm/tr46@0.0.3/node_modules/tr46, license file not found.
> ```

Yeah they don;'t seem to include the license in the package, it is in the repo though

**Comment by nguerrera** [comment]
Hmm, that's unfortunate, I guess we could curl it?

**Comment by nguerrera** [comment]
Hmm, the latest package (3.0.0) has a license.md in it as far as I can tell but we're getting 0.0.3 here.

File: common/config/rush/pnpm-lock.yaml [hunk 1]
```diff
@@ -2524,20 +2512,20 @@ packages:
       node: '>=0.8'
     resolution:
       integrity: sha512-nlLsUzgm1kfLXSXfRZMc1KLAugd4hqJHDTvc2hDIwS3mZAfMEuMbc03SujMF+GEcpaX/qboeycw6iO8JwVv2+g==
+  /tr46/0.0.3:
+    dev: false
+    resolution:
+      integrity: sha1-gYT9NH2snNwYWZLzpmIuFLnZq2o=
   /tr46/1.0.1:
     dependencies:
```
Comment (nguerrera) [inline common/config/rush/pnpm-lock.yaml:2520]: Interesting that we get v0.0.3 and v1.0.1 here.

Comment (timotheeguerin) [reply common/config/rush/pnpm-lock.yaml:2520]: found the issue, updating `node-fetch` to the latest remove the need for 0.0.3 through many nested dependencies

**Review by nguerrera — APPROVED** [review]

**Pull request merged**